poly-synth/src/poly-synth-demo.js

```diff
@@ -22,163 +22,368 @@ class PolySynthDemo {
         });
 
     let gainAttackSlider_ = new ParamController(
-        gainADSRId, this.polySynth_.setParameter.bind(this.polySynth_), {
+        containerId, this.polySynth_.setParameter.bind(this.polySynth_), {
           name: 'Attack (s)',
           id: 'gainAttack',
           type: 'range',
           min: this.polySynth_.minAttack,
           max: this.polySynth_.maxAttack,
           step: 0.01,
-          default: this.polySynth_.getParameters().gainAttack,
+          default: this.polySynth_.getParameters().gainAttack
         });
 
     let gainDecaySlider_ = new ParamController(
-        gainADSRId, this.polySynth_.setParameter.bind(this.polySynth_), {
+        containerId, this.polySynth_.setParameter.bind(this.polySynth_), {
           name: 'Decay (s)',
           id: 'gainDecay',
           type: 'range',
           min: this.polySynth_.minDecay,
           max: this.polySynth_.maxDecay,
           step: 0.01,
-          default: this.polySynth_.getParameters().gainDecay,
+          default: this.polySynth_.getParameters().gainDecay
         });
 
     let gainSustainSlider_ = new ParamController(
-        gainADSRId, this.polySynth_.setParameter.bind(this.polySynth_), {
+        containerId, this.polySynth_.setParameter.bind(this.polySynth_), {
           name: 'Sustain',
           id: 'gainSustain',
           type: 'range',
           min: this.polySynth_.minSustain,
           max: this.polySynth_.maxSustain,
           step: 0.01,
-          default: this.polySynth_.getParameters().gainSustain,
+          default: this.polySynth_.getParameters().gainSustain
         });
 
     let gainReleaseSlider_ = new ParamController(
-        gainADSRId, this.polySynth_.setParameter.bind(this.polySynth_), {
+        containerId, this.polySynth_.setParameter.bind(this.polySynth_), {
           name: 'Release (s)',
           id: 'gainRelease',
           type: 'range',
           min: this.polySynth_.minRelease,
           max: this.polySynth_.maxRelease,
           step: 0.01,
-          default: this.polySynth_.getParameters().gainRelease,
+          default: this.polySynth_.getParameters().gainRelease
         });
-
+  }
+  
+  initializeFilterEnvelopeGUI_(containerId){
     let lowPassSlider_ = new ParamController(
-        filterADSRId, this.polySynth_.setParameter.bind(this.polySynth_), {
+        containerId, this.polySynth_.setParameter.bind(this.polySynth_), {
           name: 'Cutoff (hz)',
           id: 'filterCutoff',
           type: 'range',
           min: this.polySynth_.minCutoff,
           max: this.polySynth_.maxCutoff,
           step: 1,
-          default: this.polySynth_.getParameters().filterCutoff,
+          default: this.polySynth_.getParameters().filterCutoff
         });
 
     let filterQSlider_ = new ParamController(
-        filterADSRId, this.polySynth_.setParameter.bind(this.polySynth_), {
+        containerId, this.polySynth_.setParameter.bind(this.polySynth_), {
           name: 'Q (dB)',
```

Comment:
In the spec it says: "Controls how peaked the response will be at the cutoff frequency. A large value makes the response more peaked. Please note that for this filter type, this value is not a traditional Q, but is a resonance value in decibels."